backend/authentication/auth-handler.js

```diff
@@ -74,7 +74,31 @@ module.exports.authController = async (event, context, callback) => {
             }
             break;
         }
-    //---------- LIST ALL ADMINS ----------
+        //---------- ADMIN LOGIN ----------
+        case 'adminLogin': {
+            try {
+                const cognito = new AWS.CognitoIdentityServiceProvider();
+                await cognito.adminInitiateAuth({
+                    AuthFlow: 'ADMIN_NO_SRP_AUTH',
+                    ClientId: "5sedj9nq2je1oq5banrqqcjd57",
+                    UserPoolId: "eu-central-1_wxkQGwd6a",
+                    AuthParameters: {
+                        USERNAME: event.arguments.email,
+                        PASSWORD: event.arguments.password
+                    }
+                })
+                .promise()
+                .then(response => {
```

Comment:
Instead of doing `.then...` take the response into a variable like: `const variableName = await cognito.adminInitiateAuth({ ... }).promise()`